src/app/components/faq/faq.component.ts

```diff
@@ -1,24 +1,27 @@
-import { Component, OnInit } from '@angular/core';
-import { FaqService } from '@app/_services/api/faq.service';
-import { Faq } from '@app/_models/faq';
+import {Component, OnInit} from '@angular/core';
+import {FaqService} from '@app/_services/api/faq.service';
+import {Faq} from '@app/_models/faq';
 
 @Component({
-  selector: 'app-faq',
-  templateUrl: './faq.component.html',
-  styleUrls: ['./faq.component.scss']
+    selector: 'app-faq',
+    templateUrl: './faq.component.html',
+    styleUrls: ['./faq.component.scss']
 })
 export class FaqComponent implements OnInit {
 
-  faqs: Faq[];
+    faqs: Faq[];
+    emptyObservable = true;
 
-  constructor(public faqService: FaqService) {}
+    constructor(public faqService: FaqService) {
+    }
 
-  ngOnInit(): void {
-    this.faqService
-    .getFaqs()
-    .subscribe((faqs) => {
-      this.faqs = faqs;
-    });
-  }
+    ngOnInit(): void {
+        this.faqService
+            .getFaqs()
+            .subscribe((faqs) => {
+                this.faqs = faqs;
+                this.emptyObservable = !faqs.length;
```

Comment:
You really don't need this i think. You can check If(faqs) or if(faqs.length) anywhere you need this.

Reply:
aye aye